db/seeds.rb

```diff
@@ -5,3 +5,32 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+users = [
+  {
+    name: 'User 1'
+  },
+  {
+    name: 'User 2'
+  }
+]
+
+auctions = [
+  {
+    title: 'Throne of Eldraine Booster Box',
+    description: 'New: A brand-new, unused, unopened, undamaged item (including handmade items).',
+    start_bid: 85.00,
+    current_bid: 88.50,
+    end_time: 10.days.from_now.strftime('%FT%TZ')
+  },
+  {
+    title: 'Oko, Theif of Crowns',
+    description: 'Brand new. Single card',
+    start_bid: 10.00,
+    current_bid: 11.50,
+    end_time: 4.days.from_now.strftime('%FT%TZ')
+  },
+]
+
+users.each {|user| User.create(name: user[:name])}
```

Comment:
You could just pass the hash to the `create` method, so you don't have to repeat the properties.
Same you're doing with the auctions.